index.html

```diff
@@ -0,0 +1,191 @@
+  
+  </section>
+   <section class="main-section" id="Constants"> 
+    <header>
+      Constants
+    </header>
+    <p>You can create a read-only, named constant with the const keyword. 
+        The syntax of a constant identifier is the same as for a variable identifier: 
+        it must start with a letter, 
+        underscore or dollar sign and can contain alphabetic, numeric, or underscore characters.</p>
+    <code>const PI = 3.14;</code>
+  </section>
+   <section class="main-section" id="Data_types"> 
+    <header>
+      Data types
+    </header>
+    <p>The latest ECMAScript standard defines seven data types:</p>
+
+     <ul>   
+        
+        <li>Boolean. true and false.</li>
+        <li>null. A special keyword denoting a null value.</li> 
+        <li>Number. 42 or 3.14159.</li>
+        <li>String. "Howdy"</li>
+        <li>Symbol (new in ECMAScript 2015).</li> 
+        <li>Object</li>
+
+     </ul>
+    
+  </section>
+   <section class="main-section" id="if...else_statements"> 
+    <header>
+      if...else Statements
+    </header>
+    <p>Use the if statement to execute a statement if a logical 
+        condition is true. Use the optional else clause to execute 
+        a statement if the condition is false. An if statement looks as follows:
+    </p>
+    <code>if (condition) { statement_1; } else { statement_2; }</code>   
+    <code>if (condition_1) { statement_1; } else if (condition_2) {
+        statement_2; } else if (condition_n) { statement_n; } else {
+        statement_last; }</code>
+    <code>if (x = y) { /* statements here */ }</code>
+    <code>if ((x = y)) { /* statements here */ }</code>
+  </section>
+   <section class="main-section" id="while_statements"> 
+    <header>
+      while statements
+    </header>
+    <p>A while statement executes its statements as long as a specified condition 
+        evaluates to true. A while statement looks as follows:
+    </p>
+    <code>while (condition) statement</code>
+    <code>var n = 0; var x = 0; while (n < 3) { n++; x += n; }</code>
+  </section>
+   <section class="main-section" id="Function_declaration"> 
+    <header>
+      Function declaration
+    </header>
+    <p>A function definition (also called a function declaration, 
+        or function statement) consists of the function keyword, followed by:
+    </p>
+    <ul>
+        <li>The name of the function.</li>
+        <li>A list of arguments to the function, enclosed in parentheses and separated by commas.</li>
+        <li>The JavaScript statements that define the function, enclosed in curly brackets, { }.</li>
+    </ul>
+    <code>function square(number) { return number * number; }</code>
+
+  </section>
+  <section class="main-section" id="Reference"> 
+    <header>
+      Reference
+    </header>
+    <p>All the documentation in this page is taken from Mozilla Development Network.
+    </p>
+  </section>
+ 
```

Comment:
Remove extra spaces for looks great and clean.

Reply:
- Kindly add an explanation to your comments. it might be obvious for you but not for someone else.
- Also Remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.

Reply:
Kindly remove extra spaces. It will look great.